layouts/joomla/form/field/textarea.php

```diff
@@ -62,8 +78,8 @@
 	$autocomplete,
 	$autofocus ? 'autofocus' : '',
 	$spellcheck ? '' : 'spellcheck="false"',
-	$maxlength ? $maxlength: ''
-
+	$maxlength ? $maxlength: '',
+	!empty($counterlabel)
```

Comment:
Reading this something seems off...
```suggestion
	!empty($counterlabel) ? $counterlabel : '',
```